server/sonar-server/src/main/java/org/sonar/ce/log/CeProcessLogging.java

```diff
@@ -41,7 +41,7 @@ protected void extendLogLevelConfiguration(LogLevelConfig.Builder logLevelConfig
     logLevelConfigBuilder.levelByDomain("sql", ProcessId.COMPUTE_ENGINE, LogDomain.SQL);
     logLevelConfigBuilder.levelByDomain("es", ProcessId.COMPUTE_ENGINE, LogDomain.ES);
     JMX_RMI_LOGGER_NAMES.forEach(loggerName -> logLevelConfigBuilder.levelByDomain(loggerName, ProcessId.COMPUTE_ENGINE, LogDomain.JMX));
-    MSQDRIVER_LOGGER_NAMES_TO_TURN_OFF.forEach(loggerName -> logLevelConfigBuilder.immutableLevel(loggerName, Level.OFF));
+    LOGGER_NAMES_TO_TURN_OFF.forEach(loggerName -> logLevelConfigBuilder.immutableLevel(loggerName, Level.OFF));
```

Comment:
I would name it "LOGGER_CLASSES_TO_TURN_OFF"

Reply:
Nope, loggers are referenced by string names and may not refer to classes. Using a class is a convenient helper provided by logging libraries.